tests/test-run-calculation.php

```diff
@@ -0,0 +1,273 @@
+<?php
+
+class RunCalculationTest extends Caldera_Forms_Test_Case
+{
+
+    protected $form = array(
+        '_last_updated' => 'Mon, 29 Apr 2019 16:26:35 +0000',
+        'ID' => 'calculation',
+        'cf_version' => '1.8.4',
+        'name' => 'Calculation',
+        'scroll_top' => 0,
+        'success' => 'Form has been successfully submitted. Thank you.												',
+        'db_support' => 1,
+        'pinned' => 0,
+        'hide_form' => 1,
+        'avatar_field' => '',
+        'form_ajax' => 1,
+        'custom_callback' => '',
+        'layout_grid' => 
+        array(
+          'fields' => 
+          array(
+            'fld_1892141' => '1:1',
+            'fld_93143' => '1:1',
+            'fld_7465068' => '1:1',
+            'fld_8568604' => '1:1',
+            'fld_3665187' => '1:1',
+          ),
+          'structure' => '12',
+        ),
+        'fields' => 
+        array(
+          'fld_1892141' => 
```

Comment:
@New0 You have some of the fields in the form config using fld_... for ID and some using a meaningful ID. Let's make them all meaningful and mayb use class properties. `$this->IdOdFieldThatHasValueOf1;  $this->manualCalcFieldId`